persistence/src/main/java/org/devgateway/toolkit/persistence/status/DiseasesProgress.java

```diff
@@ -0,0 +1,27 @@
+package org.devgateway.toolkit.persistence.status;
+
+import java.time.Month;
+import java.util.Map;
+import java.util.stream.Stream;
+
+/**
+ * @author Octavian Ciubotaru
+ */
+public class DiseasesProgress extends DatasetProgress {
+
+    private final Map<Month, DataEntryStatus> statuses;
+
+    public DiseasesProgress(Map<Month, DataEntryStatus> statuses) {
+        super("Situation zoo sanitaire", "Ministère de l’élevage Direction de l’Elevage (DIREL)");
```

Comment:
We should use the org name from DB instead, since it is changeable.
Also each org has indicator associated, which we can actually pass here to read indicator name + rog.

The same is for other progress instances.

Reply:
I've created a follow up ticket for this.